### ISSUE TITLE
Detector converter fails using OpenJDK

### ISSUE THREAD

**Issue by JeremyMcCormick** [JeffersonLab/hps-java#411]
There is an error with the service lookup:

```
Exception in thread "main" java.lang.IllegalArgumentException: org.lcsim.geometry.compact.converter.Converter is not an ImageIO SPI class
        at java.desktop/javax.imageio.spi.ServiceRegistry.checkClassAllowed(ServiceRegistry.java:722)
        at java.desktop/javax.imageio.spi.ServiceRegistry.lookupProviders(ServiceRegistry.java:170)
        at org.hps.detector.DetectorConverter.getServices(DetectorConverter.java:65)
        at org.hps.detector.DetectorConverter.getConverters(DetectorConverter.java:57)
        at org.hps.detector.DetectorConverter.run(DetectorConverter.java:88)
        at org.hps.detector.DetectorConverter.main(DetectorConverter.java:53)
```

This problem should be fixable by using this class instead:

https://docs.oracle.com/javase/6/docs/api/java/util/ServiceLoader.html

**Comment by JeremyMcCormick** [comment]
This actually needs to be fixed in lcsim, not hps-java.

**Comment by JeremyMcCormick** [comment]
Fixed by PR #463.
